wire-library/wire-protoc/src/main/java/com/squareup/wire/protocwire/WireGenerator.kt

```diff
@@ -320,10 +347,10 @@ private data class OptionValueAndKind(val value: Any, val kind: OptionElement.Ki
 private data class SourceCodeInfo(val comment: String, val loc: Location)
 
 private class SourceCodeHelper(
-  val locs: Map<List<Int>, DescriptorProtos.SourceCodeInfo.Location>,
-  val baseLoc: Location
+  fd: DescriptorProtos.FileDescriptorProto
```

Comment:
Does this not result in a field named `fd` on this class? I was trying to avoid that, though I suppose it doesn't matter much...

Reply:
This wont. It's a little fun because if you don't declare `val` it's only accessible in the constructor or init blocks. I figured this was what you were going for but I was like trying to rationalize how to use this class and went ahead to changing it